common/eth2/src/lighthouse/block_rewards_v2.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
+
+// Details about the rewards for a single block
+// All rewards in GWei
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+// TODO: Discuss name BlockRewardsV2
+pub struct BlockRewardsV2 {
```

Comment:
I think `StandardBlockRewards` would be suitable, to distinguish it from the Lighthouse-specific API